### PR TITLE
feat(tracking): add additional home page telemetry

### PR DESCRIPTION
**Pull request by aditya-radhakrishnan**
## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [x] Links to related issues (if applicable)
- [x] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [x] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: datahub-web-react/src/app/home/HomePage.tsx [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import React from 'react';
 import { HomePageHeader } from './HomePageHeader';
 import { HomePageBody } from './HomePageBody';
+import analytics, { EventType } from '../analytics';
 
 export const HomePage = () => {
+    analytics.event({ type: EventType.PageViewEvent, originPath: window.location.pathname });
```
Comment (gabe-lyons) [inline datahub-web-react/src/app/home/HomePage.tsx:7]: let's make a homepage-specific EventType

File: datahub-web-react/src/app/search/BrowseEntityCard.tsx [hunk 2]
```diff
@@ -5,20 +5,32 @@ import { PageRoutes } from '../../conf/Global';
...
-    const url =
-        entityType === EntityType.GlossaryTerm
-            ? PageRoutes.GLOSSARY
-            : `${PageRoutes.BROWSE}/${entityRegistry.getPathName(entityType)}`;
+    const isGlossaryEntityCard = entityType === EntityType.GlossaryTerm;
+    const entityPathName = entityRegistry.getPathName(entityType);
+    const url = isGlossaryEntityCard ? PageRoutes.GLOSSARY : `${PageRoutes.BROWSE}/${entityPathName}`;
+    const onBrowseEntityCardClick = () => {
+        analytics.event({
+            type: EventType.BrowseResultClickEvent,
```
Comment (gabe-lyons) [inline datahub-web-react/src/app/search/BrowseEntityCard.tsx:18]: lets make this homepage-browse specific

File: datahub-web-react/src/app/home/HomePage.tsx (hunk 1 above)
Comment (aditya-radhakrishnan) [reply datahub-web-react/src/app/home/HomePage.tsx]: Added a HomePageViewEvent!

File: datahub-web-react/src/app/search/BrowseEntityCard.tsx (hunk 2 above)
Comment (aditya-radhakrishnan) [reply datahub-web-react/src/app/search/BrowseEntityCard.tsx]: Added a HomePageBrowseResultClickEvent!

**Review by gabe-lyons — APPROVED** [review]

**Pull request merged**